sdk/core/ci.yml

```diff
@@ -40,4 +40,9 @@ stages:
     ArtifactName: packages
     Artifacts:
     - name: Azure.Core
-      safeName: AzureCore
\ No newline at end of file
+      safeName: AzureCore
+    - name: Microsoft.Extensions.Azure
+      safeName: MicrosoftExtensionsAzure
+      options:
+        skipTagRepository: true
```

Comment:
Does this mean the only think we want is to enable publishing of docs? Wouldn't we also need to publish the library package also at some point?

Reply:
@chidozieononiwu was saying that we don't for some reason. But I'm not across the background. This PR was just an example of how to use the properties. Now that Dozie has other PRs that use the properties I'll close this one down.

Reply:
I'm not sure I like having persisted logic that always disables tagging and publishing of the package. If it is just for publishing of the docs we should do that as a one-off.

Reply:
I've made tagging mandatory in the release stage however I think this will create a bit of a problem for doc publishing. @chidozieononiwu be aware that the option.skipTagRepository option if you have it will be ignored.